File: 344-reverse-string/reverse-string.cpp
class Solution {
public:
    void reverseString(vector<char>& s) 
    {
        int n = s.size();
        int start = 0;
        int end = n - 1;
        while(start < end)
        {
            char temp = s[end];
            s[end] = s[start];
            s[start] = temp;
            start++;
            end--;
        }   
    }
};

File: 344-reverse-string/reverse-string.js
/**
 * @param {character[]} s
 * @return {void} Do not return anything, modify s in-place instead.
 */
var reverseString = function(s) {
    n = s.length;
    for(let i = 0; i < n/2; i++) {
        let temp = s[i];
        s[i] = s[n-i-1];
        s[n-i-1] = temp;
    }
    return s;
};